synapse/handlers/e2e_keys.py

```diff
@@ -196,7 +220,73 @@ def do_remote_query(destination):
             ).addErrback(unwrapFirstError)
         )
 
-        return {"device_keys": results, "failures": failures}
+        ret = {"device_keys": results, "failures": failures}
+
+        for key, value in iteritems(cross_signing_keys):
+            ret[key + "_keys"] = value
```

Comment:
could we make the keys in `cross_signing_keys` match those in the final result to avoid this magic (hence: `master_keys`, `self_signing_keys`, etc)?